spec/deck_spec.rb

```diff
@@ -0,0 +1,47 @@
+require './lib/deck.rb'
+require './lib/card.rb'
+
+RSpec.describe Deck do
+  it 'exists' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that  is 697.5° clockwise from due north?", "North north west", :STEM)
+
+     cards = [card_1, card_2, card_3]
+     deck = Deck.new(cards)
+    expect(deck).to be_instance_of(Deck)
+  end
+
+  it 'cards in deck' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that  is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    expect(deck).to be_instance_of(Deck)
+  end
+
+  it 'card count in deck' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that  is 697.5° clockwise from due north?", "North north west", :STEM)
+
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    expect(cards.count).to eq(3)
```

Comment:
with this assertion, you're not testing the cards attribute that's on your deck object. You're just testing the cards variable you have declared on line 30 which has nothing to do with your deck object. So, instead, you could write an assertion like this: 

`expect(deck.cards).to eq(cards)`